langchain/agents/tools/requests.ts

```diff
@@ -0,0 +1,69 @@
+import { Tool, Toolkit } from "./base";
+
+type headers = { [key: string]: string };
+
+interface RequestTool {
+  headers: headers;
+}
+
+export class RequestsGetTool extends Tool implements RequestTool {
+  name = "requests_get";
+
+  headers: headers;
+
+  constructor(headers?: headers) {
```

Comment:
you can use TS "parameter properties" with a default value, like this `constructor(public headers?: Headers = {})`